template.go

```diff
@@ -149,8 +152,12 @@ const (
 				defer mm_atomic.AddUint64(&m.after{{$method.Name}}Counter, 1)
 
 				{{if $method.HasParams}}
+					// Record call args
+					params := &{{$mock}}{{$method.Name}}Params{ {{$method.ParamsNames}} }
+					m.{{$method.Name}}Mock.callArgs = append(m.{{$method.Name}}Mock.callArgs, params)
```

Comment:
Append is not concurrent safe so if the same mock is called by multiple goroutines we're going to have race here.

Reply:
Fixed